scripts/Copyright-Checker/copyright-checker.sh

```diff
@@ -62,20 +62,35 @@ function is_copyright_valid() {
   local file_extension="${file##*.}"
   local offset=1
 
-  # since PSR-12 the php files will contain the copyright license as
-  # document-level comment, which starts on line 3.
-  if [ "php" = "${file_extension}" ]; then
-    offset=3
-  fi
+  is_ui_example ${file}
+  local is_example=${?}
 
-  for copyright_line in "${COPYRIGHT_LINES[@]}"; do
-    local line_to_check="$(sed "${offset}q;d" "${file}")"
-    if ! [ "${copyright_line}" = "${line_to_check}" ]; then
-      return 1
+  if [ 1 -eq ${is_example} ]; then
+    if [ "php" = ${file_extension} ]; then
+      offset=3
     fi
-
-    offset=$((1 + ${offset}))
-  done
+    for copyright_line in "${COPYRIGHT_LINES[@]}"; do
+      local line_to_check="$(sed "${offset}q;d" "${file}")"
+      if ! [ "${copyright_line}" = "${line_to_check}" ]; then
+        return 1
+      fi
+
+      offset=$((1 + ${offset}))
+    done
+  else
+    local lines=$(wc -l < ${file});
+    while [ ${offset} -lt ${lines} ]
+    do
+      local line_to_check="$(echo "$(sed "${offset}q;d" ${file})" | xargs)"
+      if [[ "${line_to_check}" == "namespace "* ]]; then
+        return 0
+      fi
+      if [[ "${line_to_check}" == "/*"* || "${line_to_check}" == "//"* || "${line_to_check}" == "#"* ]]; then
```

Comment:
```suggestion
      if echo "$line" | grep -q -e '^/\*' -e '^//' -e '^#'; then
```
Same as above, please ignore if you disagree.